src/config/index.js

```diff
@@ -1,10 +1,3 @@
-import qs from 'query-string';
-
-const hasFeatureFlagEnabled = (featureFlag) => {
```

Comment:
I'm mixed on if it's worth keeping `hasFeatureFlagEnabled` around or not. In my head the pro's and con's are
PROS:
- Doesn't impede/break anything to keep it
- if an engineer is testing something, say on stage, this offers an easy way to toggle the features on the page without having to create a PR and add a change to edx-internal
CONS:
- Allows devs to be slightly lazy in that they're not forced to enable features on stage
- Current implementation means that the user has to add the query parameters _every_. _single_. _time_. they open/load the page

Overall I think I lean towards removing but want to put the question out there if there's any merit in keeping it around.

Reply:
yeah I think it's useful while you are still not released the feature fully and turned on in edx-internal by default. once it's turned on the feature should not need to be toggled via browser. I think that is useful while we are still off, and maube want to test by turning it on in the browser...

Reply:
yeah I believe @binodpant also brought up a similarish concern: `but don't we still need that setup in case we want to do that for a future flag?`

And yeah the flag function is still definitely a way to not have to make a separate PR in order to test a change in this repo.
However, to your con about devs not enabling the feature in stage: I see the default enabling of features in stage as an additional layer to catch bugs. You might test the specific feature when enabling the flag, but as others are pushing changes to stage, we can inherently test the feature's interaction with other areas of the app and other users' changes by making it enabled for *everyone*.

The real pro I see to having it there is that if your feature completely breaks something, it is turned off automatically for other devs. That's a pretty big pro, but at the same time, why/how often are we merging and thus shipping completely broken code that doesn't even do what was intended and crashes the service such that everyone is affected? I feel like that is something to alert everyone on because uhhhhhh I don't want that code in prod.

Anyways, I know it doesn't come across here, but I am also not opposed to keeping the feature around. I just also don't see a _necessity_ in it and thus just went about deleting!all!the!things!

Reply:
yeah I would say let's leave it in? if we end up not using it at all yeah we can scrap it.. I just see it being useful sometimes... (also it's quite easy to add if/when needed so not worried either way).